rasa/model.py

```diff
@@ -128,16 +128,19 @@ def should_retrain_nlu(self) -> bool:
         return self.force_training or self.nlu
 
 
-def get_model(model_path: Text = DEFAULT_MODELS_PATH) -> TempDirectoryPath:
-    """Get a model and unpack it. Raises a `ModelNotFound` exception if
-    no model could be found at the provided path.
+def get_local_model(model_path: Text = DEFAULT_MODELS_PATH) -> Text:
+    """Verifies that a model path exists.
```

Comment:
I think we need to adapt that one, too. How about:
```suggestion
    """Returns verified path to local model archive.
```